exonum/src/proto/schema/exonum/runtime/lifecycle.proto

```diff
@@ -47,8 +47,8 @@ message GenesisConfig {
 message ArtifactState {
   // Status of an artifact deployment.
   enum Status {
-    // Reserved for the further usage.
-    NONE = 0;
+    // The artifact is pending unload.
+    UNLOADING = 0;
     // The artifact is pending deployment.
     PENDING = 1;
```

Comment:
UNLOADING and PENDING looks a little bit inconsistend together. I consider that we should rename PENDING to DEPLOYING or LOADING.